assets/sources/base_client.py

```diff
@@ -39,36 +40,43 @@ def get(self, url, **kwargs):
                 return r
             return False
 
+
 results = dict()
-def cache(func):
 
+
+def cache(func):
     def wrapper(*args, **kwargs):
         if kwargs["cache_id"] in results:
             return results[kwargs["cache_id"]]
-        result= func(*args)
+        result = func(*args)
         results[kwargs["cache_id"]] = result
         return result
 
     return wrapper
 
+
 @cache
 def get_selenium_webdriver(cache_id=None):
     chrome_options = Options()
     chrome_options.add_argument("--headless")
     return webdriver.Chrome(chrome_options=chrome_options)
 
+
 @cache
 def killed(cache_id=None):
     return True
 
+
 class BaseSeleniumClient:
     def __init__(self):
         # google-chrome is installed on flip redhat osu servers
         # if you want to use something else locally, you will need
         # to configure this yourself
         chrome_options = Options()
         chrome_options.add_argument("--headless")
-        self.selenium = get_selenium_webdriver(cache_id="base_client_selenium_webdriver")
+        self.selenium = get_selenium_webdriver(
+            cache_id="base_client_selenium_webdriver"
+        )
         self.pid_proc = self.selenium.service.process.pid
 
     def __del__(self):
```

Comment:
If anyone has any ideas on how to do this part better, I'm all ears, as I'm at my wits end with it.

Reply:
Is this in regards to killing the selenium process?